modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,46 @@
+
+The HLV metadata takes up `109 + 40N` bytes of space per document, where N is the number of buckets mutating the document in the replication topology. 
+
+NOTE: The size of the HLV metadata must remain constant as long as your replication topology is constant. However, the HLV data accumulation occurs when the replication topology changes and the document copy goes through different clusters, for example, when being restored to a new cluster.
+
+To remove the accumulated or outdated HLV data, the HLV metadata is pruned periodically. You can control the pruning frequency by setting the bucket property `versionPruningWindowHrs`.
+
+IMPORTANT: This is an important limitation to the _XDCR Active-Active with Sync Gateway_ feature. If you use the user created extended attributes (xattrs) in your documents, and you have more than 10 user xattrs in a document, then you cannot use the feature _XDCR Active-Active with Sync Gateway_. This is due to an internal limitation of managing extended attributes in a document. If you try to use the feature _XDCR Active-Active with Sync Gateway_ when you have more than 10 user xattrs in your document, the XDCR replication **silently skips** replicating the document that has more than 10 user xattrs. As a result, the source and target data in the replication skipped documents will not be consistent. The only way you will know this skip occured is because the Prometheus stat **subdoc_cmd_docs_skipped** will be incremented and the document will _not_ be consistent between the target and source. So, if your documents have more than 10 user xattrs, do _not_ enable the property `enableCrossClusterVersioning` with a purpose to use the _XDCR Active-Active with Sync Gateway_ feature.
+
+[#modify-enablecrossclusterversioning]
+== Modify the bucket property enableCrossClusterVersioning
+
+You can modify the bucket property `enableCrossClusterVersioning` through REST API. For information about modifying `enableCrossClusterVersioning` through REST API, see xref:rest-api:rest-bucket-create.adoc#example-enablecrossclusterversioning-edit[Example: Turning on enableCrossClusterVersioning, when Editing].
+
+[#version-pruning-window-hrs]
+== versionPruningWindowHrs
+
+`versionPruningWindowHrs` is a bucket property, which controls the pruning frequency of the HLV metadata. The default value of versionPruningWindowHrs is 720 hours (30 days), which means that any HLV data older than 720 hours is pruned to remove the outdated entries. 
+
+NOTE: `versionPruningWindowHrs` must be set to the same value for all buckets in an XDCR replication topology.
+
+[#manage-version-pruning-window-hrs]
+=== Manage the bucket property versionPruningWindowHrs
+
+You can manage the bucket property `versionPruningWindowHrs` through REST API as follows:
+
+* For information about creating the bucket property `versionPruningWindowHrs` through REST API, see xref:rest-api:rest-bucket-create.adoc#curl-syntax[Curl Syntax].
```

Comment:
You don't need this info about creating the bucket property `versionPruningWindowHrs` -- it's always exists when a bucket is created.  So, remove this sentence.